From e9bcab0f8fab09f14a63e2685613a5069ff6b07e Mon Sep 17 00:00:00 2001
From: Mario Limonciello <mario.limonciello@dell.com>
Date: Mon, 17 Jul 2017 10:49:57 -0500
Subject: [PATCH] Default to "en" for UEFI capsule graphics

"C" is not a supported locale code per ISO 639-2
see https://lintian.debian.org/tags/unknown-locale-code.html
for more information
---
 po/make-images | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/po/make-images b/po/make-images
index 81c5533..5047fbe 100755
--- a/po/make-images
+++ b/po/make-images
@@ -72,7 +72,7 @@ class Rasterizer:
             if x == 'hi':
                 continue
             yield x
-        yield "C"
+        yield "en"
 
     def render(self):
         """ Do the work... """
@@ -167,7 +167,7 @@ class Rasterizer:
                 'directory': self.directory,
                 'language': lang,
             }
-            if lang == 'C':
+            if lang == 'en':
                 string = self.text
             else:
                 tl = gettext.translation(self.domain, self.modir,
-- 
2.7.4

